src/test/java/staffconnect/testutil/TypicalPersons.java

```diff
@@ -100,6 +107,7 @@ public static StaffBook getTypicalStaffBook() {
     }
 
     public static List<Person> getTypicalPersons() {
-        return new ArrayList<>(Arrays.asList(ALICE, BENSON, CARL, CLARA, DANIEL, ELLE, FIONA, GEORGE, KAFKA, NATASHA));
+        return new ArrayList<>(Arrays.asList(ALICE, BENSON, CARL, CLARA, DANIEL, ELLE, FIONA, GEORGE, KAFKA, NATASHA,
+                LEONARDO, MICHAEL));
     }
```

Comment:
ordering here is not in alphabetical order, next time we can edit this too

Reply:
Noted: #97